sinusbot_installer.sh

```diff
@@ -663,7 +663,7 @@ greenMessage "Packages installed"!
 if [[ $VIRTUALIZATION_TYPE == "openvz" ]]; then
   redMessage "You're using OpenVZ virtualization. You can't set your time, maybe it works but there is no guarantee. Skipping this part..."
 else
-  if [ -f /etc/centos-release ] || [ $(cat /etc/*release | grep "DISTRIB_ID=" | sed 's/DISTRIB_ID=//g') ]; then
+  if [[ -f /etc/centos-release ]] || [ $(cat /etc/*release | grep "DISTRIB_ID=" | sed 's/DISTRIB_ID=//g') ]; then
```

Comment:
Is there a reason why you only do that on the first condition and not on the rest? (applies to multiple lines)

Reply:
Only file checks are neccessary to use double square brackets.
All other condition may be break.

I tested this enhancement on Ubuntu 18.04, 19.04, Debian 9-10 and CentOS7 without problems.